packages/pn-personafisica-login/src/pages/login/Login.tsx

```diff
@@ -30,6 +31,9 @@ const Login = () => {
   const [showIDPS, setShowIDPS] = useState(false);
   const { t, i18n } = useTranslation(['login', 'notifiche']);
   const isMobile = useIsMobile();
+  const [params] = useSearchParams();
+
+  sessionStorage.setItem('redirectUrl', params.get('path') ?? '');
```

Comment:
uniformiamo l'accesso al sessionStorage usando il metodo storageOpsBuilder in storage.utility.ts (vedi esempio in storage.ts)

Reply:
Aggiunto l'oggetto storageOriginOps e utilizzato nelle pagine Login e Redirect